core/builder/src/main/java/io/quarkus/builder/BuildException.java

```diff
@@ -12,17 +13,52 @@
 public class BuildException extends Exception {
```

Comment:
Extend from Qlue's `ChainBuildException` ?

Reply:
I was hesitant to do that because it would change the hierarchy and I'm not sure if it would be beneficial in any way.